src/subdatatable/subdatatable.jl

```diff
@@ -93,27 +93,22 @@ function SubDataTable(parent::DataTable, row::Integer)
     return SubDataTable(parent, [Int(row)])
 end
 
-function SubDataTable{S <: Integer}(parent::DataTable, rows::AbstractVector{S})
+function SubDataTable(parent::DataTable, rows::AbstractVector{<:Union{Integer, Null}})
     return SubDataTable(parent, convert(Vector{Int}, rows))
 end
 
 function SubDataTable(parent::DataTable, rows::AbstractVector{Bool})
     return SubDataTable(parent, find(rows))
 end
 
-function SubDataTable{T<:Integer}(sdt::SubDataTable, rowinds::Union{T, AbstractVector{T}})
+function SubDataTable{T<:Union{Integer, Null}}(sdt::SubDataTable, rowinds::Union{T, AbstractVector{T}})
```

Comment:
I had left a comment in my first review but we didn't answer it: are nulls actually supported in `rowindes`? The old code didn't accept `Nullable`.